api_handlers.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2017 Google Inc.
```

Comment:
@jeffcarp do we have a process for Go tests? I don't see any in this PR.

Reply:
@mdittmer nothing formalized yet. Some basic tests would be great. Here's a good example @lukebjerring landed recently: https://github.com/w3c/wptdashboard/blob/master/json_redirect_handler_test.go

Reply:
Will look at re-jigging and adding tests in a follow-up - this PR is already big enough.